docs/doc-iothub-storage-private.md

```diff
@@ -0,0 +1,150 @@
+
+Azure Application Gateway will act as the entry point and load balancer for requests forwarded to the private endpoint of the Azure Storage account. We will use a Public IP address that can be mapped to an A record or CNAME at your custom DNS provider.
+
+To configure Azure Application Gateway, follow one of the available [Quickstarts](https://learn.microsoft.com/azure/application-gateway/quick-create-portal) in the official documentation. After creation of the Application Gateway resource, let's update the settings to reflect a correct setup for connecting to the Azure Storage account.
+
+In order to be able to configure the Application Gateway ensure:
+
+* Have configured a Public IP address for the front-end configuration
+* Have access to a custom DNS provider to configure your custom DNS or sub-domain entry
+  * Use a CNAME pointing to the DNS of the Public IP address in Azure. This will be in the form of `xxx.region.cloudapp.azure.com`
+* Have an SSL certificate in the form of .PFX file for the custom domain.
+
+### Backend pool
+
+The backend pool is the endpoint to which the gateway will send traffic to. Because of private DNS resolution, you can use the normal Blob endpoint as it will resolve to the <accountname>.privatelink.blob.core.windows.net which points to the private endpoint of the Storage account.
+
+<img alt="Application Gateway Backend Pool" src="assets/hub-spoke-storage-iot-appgw-backendpool.png" width="500" />
+
+### Backend setting
+
+Create the backend setting to send traffic to the backend pool using HTTPS.
+
+<img alt="Application Gateway Backend Settings" src="assets/hub-spoke-storage-iot-appgw-backendsetting.png" width="500" />
+
+### Health probe
+
+Create a new health probe pointing to the Azure Storage account. It is important to note that the HTTP response status code match list must include 400. This is because the probe is pointing to `/` root of the Blob endpoint which will be returning a 400 status code.
+
+<img alt="Application Gateway Health probe configuration" src="assets/hub-spoke-storage-iot-appgw-healthp.png" width="500" />
+
+### HTTPS listener
+
+The listener is the entry point in the Application Gateway, and this is where you will need to setup TLS certificate settings. Ensure you have an SSL certificate for the domain name you are mapping your domain to.
+
+<img alt="Application Gateway HTTPS listener settings" src="assets/hub-spoke-storage-iot-appgw-listener.png" width="500" />
+
+### Application gateway Rule
+
+Finally ensure the rule is set to the new listener, backend pool target and backend settings.
+
+<img alt="Application Gateway Routing Rule" src="assets/hub-spoke-storage-iot-appgw-rule.png" width="500" />
+
+## Hub-Spoke networking topology with Azure Firewall
+
+In the case you would like to setup Hub-Spoke networking topology, routing tables will ensure traffic from different Virtual Networks will  pass through  Azure Firewall for traffic inspection.
```

Comment:
check "routing tables will ensure traffic from different Virtual Networks will  pass through  Azure Firewall for traffic inspection" --> duplicated use of "will" and few blanks around "pass through"

Reply:
Using 'tunneling' through, better